lib/membrane_element_udp/source.ex

```diff
@@ -46,6 +44,11 @@ defmodule Membrane.Element.UDP.Source do
     {:ok, %{local_socket: socket}}
   end
 
+  @impl true
+  def handle_prepared_to_playing(_ctx, state) do
+    {{:ok, caps: {:output, %Stream{type: :packet_stream}}}, state}
+  end
```

Comment:
In other plugins we do this in `handle_caps`

Reply:
It's a source, there's no handle_caps ;)

Reply:
true :man_facepalming: